torch/nn/modules/linear.py

```diff
@@ -108,6 +108,51 @@ def extra_repr(self) -> str:
         )
 
 
+class Bias(Module):
+    r"""Applies a bias to the incoming data: :math:`y = x + b`
+
+    This module supports :ref:`TensorFloat32<tf32_on_ampere>`.
+
+    Args:
+        num_features: size of its vector parameter
+
+    Shape:
+        - Input: :math:`(*, H_{in})` where :math:`*` means any number of
+          dimensions including none and :math:`H_{in} = \text{num\_features}`.
+        - Output: :math:`(*, H_{in})` where the dimensions
+          are the same shape as the input and :math:`H_{in} = \text{num\_features}`.
+
+    Attributes:
+        bias:   the learnable bias of the module of shape :math:`(\text{out\_features})`.
+                If :attr:`bias` is ``True``, the values are initialized from
+                :math:`\mathcal{U}(-\sqrt{k}, \sqrt{k})` where
+                :math:`k = \frac{1}{\text{in\_features}}`
```

Comment:
nit: docs should indicate that params are initialized from a standard normal distribution

Reply:
done